middlewares/settings.middleware.js

```diff
@@ -19,18 +22,8 @@ const Settings = require("../models/settings.model");
  * @description Put relevent settings attributes into settingsDetails
  */
 function parsePatch(req, res, next) {
```

Comment:
I am in favor of removing this parsePatch completely. Instead of calling `middleware.settings.parsePatch`, we would call `Services.ParsePatch.parsePatch(Model.Settings, "settingDetails")` in the route.

Reply:
that makes a lot more sense (Y)